commands/applycmd.go

```diff
@@ -35,25 +40,56 @@ func GetApplyRunner(provider provider.Provider, loader manifestreader.ManifestLo
 type ApplyRunnerWrapper struct {
```

Comment:
Do we still need the wrapper now that we have moved the cobra commands from cli-utils into the kpt repo?

Reply:
I think we can get rid of the wrapper. I will do that in a follow-on PR.